rag-text2sql/app.py

```
@@ -0,0 +1,482 @@
+st.markdown("""
+This app demonstrates a custom agent that can query either a LlamaCloud index for RAG-based retrieval 
+or a SQL database as a tool. The example uses Wikipedia pages of US cities and a SQL database 
+of their populations and states.
+
+**NOTE**: Any Text-to-SQL application should be aware that executing arbitrary SQL queries can be a security risk.
+""")
+
+# Show image if available
+try:
+    st.image("llamacloud_sql_router_img.png", use_container_width=True)
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)